message_ix_models/report/legacy/__init__.py

```diff
@@ -0,0 +1 @@
+from .iamc_report_hackathon import report as legacy_report
```

Comment:
We could perhaps omit this, since the intended entry-point for users is `message_ix_models.report.report()`.